### PR TITLE
histidine as aromatic residue

### PR DESCRIPTION
**Pull request by comradeameba**
Modifications to include histidine as aromatic residue in pi-cation, pi-stacking and ti-stacking interactions

### CODE REVIEW

File: contact_calc/aromatics.py [hunk 1]
```diff
@@ -90,7 +91,14 @@ def get_aromatic_triplet(molid, frame, aatom, index_to_atom):
     residue_to_atom_names = {
         "PHE": "CG CE1 CE2",
         "TRP": "CD2 CZ2 CZ3",
-        "TYR": "CG CE1 CE2"
+        "TYR": "CG CE1 CE2",
+        "HIS": "CG CE1 CD2 NE2 NE1",
```
[the remark this answers is not in the document]
Comment (comradeameba) [reply contact_calc/aromatics.py]: As far as I've seen, these three atoms are used to find out the center of the plane. Wouldn't it be wrongly calculated, in a pentagonal ring, with only three atoms?

File: contact_calc/pi_cation.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 aromatic_phe = "((resname PHE) and (name CG CE1 CE2))"
 aromatic_trp = "((resname TRP) and (name CD2 CZ2 CZ3))"
 aromatic_tyr = "((resname TYR) and (name CG CE1 CE2))"
+aromatic_his = "((resname HIS HSD HSE HSP HIE HIP HID) and (name CG CE1 CD2 NE2 NE1))"
```
[the remark this answers is not in the document]
Comment (comradeameba) [reply contact_calc/pi_cation.py]: Done

File: contact_calc/aromatics.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (comradeameba) [reply contact_calc/aromatics.py]: Ok now I've realized the "get_aromatic_triplet" thing. 
It is done

[the remark this answers is not in the document]
Comment (comradeameba) [reply contact_calc/aromatics.py]: Done

File: contact_calc/pi_cation.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (comradeameba) [reply contact_calc/pi_cation.py]: Done

**Comment by RasmusFonseca** [comment]
Wonderful. Thank you so much David!

**Pull request merged**